omigami/tests/conftest.py

```diff
@@ -49,15 +63,15 @@ def sample_response():
 
 @pytest.fixture(scope="module")
 def ms2deepscore_client():
-    token = config["login"]["dev"]["token"].get()
-    client = MS2DeepScore(token)
+    _set_credentials_and_auth_for_tests()
+    client = MS2DeepScore()
     client._PREDICT_ENDPOINT_BASE = "https://mlops.datarevenue.com/seldon/seldon/ms2deepscore-{ion_mode}/api/v0.1/predictions"
```

Comment:
Should we change this URL?

Reply:
This is for tests, so its ok to point to dev cluster, unless you mean something else